store/session.go

```diff
@@ -0,0 +1,171 @@
+package store
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/verifa/coastline/ent"
+	"github.com/verifa/coastline/ent/group"
+	entSession "github.com/verifa/coastline/ent/session"
+	"github.com/verifa/coastline/ent/user"
+	"github.com/verifa/coastline/server/session"
+)
+
+func (s *Store) NewSession(clientID string, claims *session.UserClaims) (uuid.UUID, error) {
+
+	userDB, err := s.createUpdateUser(clientID, claims)
```

Comment:
Clean up variable naming. Currently used elsewhere is `dbUser` not `userDB`. Keep it consistent